packages/rum-core/src/domain/rumEventsCollection/action/trackClickActions.ts

```diff
@@ -75,7 +75,6 @@ export function trackClickActions(
     history.reset()
   })
 
-  lifeCycle.subscribe(LifeCycleEventType.BEFORE_UNLOAD, stopClickChain)
   lifeCycle.subscribe(LifeCycleEventType.VIEW_ENDED, stopClickChain)
```

Comment:
Note: the `BEFORE_UNLOAD` line have been removed intentionally here: because `VIEW_ENDED` is also triggered on `BEFORE_UNLOAD`, having both was unnecessary.